Sources/Action.swift

```diff
@@ -37,13 +44,17 @@ public final class Action<Input, Output, Error: Swift.Error> {
 	///
 	/// In other words, this sends every value from every unit of work that the `Action`
 	/// executes.
-	public let values: Signal<Output, NoError>
+	public private(set) lazy var values: Signal<Output, NoError> = { [unowned self] in
```

Comment:
Reverted in https://github.com/ReactiveCocoa/ReactiveSwift/pull/400/commits/4acaf81a54c9da4940b8484cf745f73ae3fd14d7.